CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to c2cgeoportal
+
+Thanks for your interest in contributing to a GeoMapFish project.
+
+## Asking Questions
+
+Please ask questions about using the library on [Stack Overflow using the tag 'openlayers']
```

Comment:
Maybe not "openlayers" ;)
Is someone of the team monitoring SW?

Reply:
To be discussed, @ybolognini